### DISCUSSION TITLE
Using Claims History for In-Network File when unable to extract rates (i.e. percent of billed charges)

### DISCUSSION THREAD

**Discussion by dawndrake58** [CMSgov/price-transparency-guide#197] [Q&A] [answered]
There are a handful of pricing methods that do not lie within the system and are not accessible to extract.  We also have a large number of percent of billed charges contracts for outpatient hospital services and the like.  In addition, PMPM is fully a financial operation.  In these instances (without further guidance from CMS or a change in the schema) can we use claim history to extract an average of previously paid services?

**Comment by ericsilvertx** [comment]
I am in the same situation and would need to use claim history for in network file.  It means however that since we can only use historical data we can not generate a pricing info from all possible providers and service code.

We are thinking of perhaps used the last or the max allowed amount for the service for the provider and the tin.

**Comment by shaselton-usds** [comment, marked as answer]
The 'percent of billed charges' negotiated rate allows for a percentage reporting.  The [`negotiated_type`](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-price-object) allows for the value of 'percentage' and the value that would be entered into the `negotiated_rate` would be the percentage number (whole number).
